share/functions/__fish_is_first_token.fish

```diff
@@ -1,15 +1,4 @@
 
 function __fish_is_first_token -d 'Test if no non-switch argument has been specified yet'
```

Comment:
Is this used anywhere in our completions? If it is, those should be updated to call `fish_is_nth_token 1` directly.